business/mesh.go

```diff
@@ -372,6 +374,11 @@ func (in *MeshService) resolveKialiNetwork() (string, error) {
 		// This setup is OK, it's just that it's not within our multi-cluster assumptions.
 		log.Warningf("Cannot resolve the network ID of the cluster where Kiali is hosted: cannot get the sidecar injector config map :%v", err)
 		return "", nil
+	} else if istioSidecarConfig == nil {
+		// Don't return an error, as this may mean that Kiali is not installed along the control plane.
```

Comment:
see my comment in the backport PR
```suggestion
		// Don't return an error, as this may mean that Kiali is not installed in the control plane namespace.
```

Reply:
Let's discuss in the backport PR. I'll change the comment in another work, if needed.